src/Tizen.Multimedia.Remoting/MediaController/EnumExtensions.cs

```diff
@@ -21,79 +21,105 @@ namespace Tizen.Multimedia.Remoting
+                case MediaControllerNativePlaybackAction.Rewind: return MediaControlPlaybackCommand.Rewind;
+                case MediaControllerNativePlaybackAction.Toggle: return MediaControlPlaybackCommand.Toggle;
             }
 
-            Debug.Fail($"Not supported code for playback command{code}.");
+            Debug.Fail($"Not supported code for playback command{nativeAction}.");
             return MediaControlPlaybackCommand.Play;
         }
 
-        internal static MediaControllerPlaybackCode ToCode(this MediaControlPlaybackCommand command)
+        internal static MediaControllerNativePlaybackAction ToNative(this MediaControlPlaybackCommand command)
         {
             switch (command)
             {
-                case MediaControlPlaybackCommand.Play: return MediaControllerPlaybackCode.Play;
-                case MediaControlPlaybackCommand.Pause: return MediaControllerPlaybackCode.Pause;
-                case MediaControlPlaybackCommand.Stop: return MediaControllerPlaybackCode.Stop;
-                case MediaControlPlaybackCommand.Next: return MediaControllerPlaybackCode.Next;
-                case MediaControlPlaybackCommand.Previous: return MediaControllerPlaybackCode.Prev;
-                case MediaControlPlaybackCommand.FastForward: return MediaControllerPlaybackCode.FastForward;
-                case MediaControlPlaybackCommand.Rewind: return MediaControllerPlaybackCode.Rewind;
+                case MediaControlPlaybackCommand.Play: return MediaControllerNativePlaybackAction.Play;
+                case MediaControlPlaybackCommand.Pause: return MediaControllerNativePlaybackAction.Pause;
+                case MediaControlPlaybackCommand.Stop: return MediaControllerNativePlaybackAction.Stop;
+                case MediaControlPlaybackCommand.Next: return MediaControllerNativePlaybackAction.Next;
+                case MediaControlPlaybackCommand.Previous: return MediaControllerNativePlaybackAction.Prev;
+                case MediaControlPlaybackCommand.FastForward: return MediaControllerNativePlaybackAction.FastForward;
+                case MediaControlPlaybackCommand.Rewind: return MediaControllerNativePlaybackAction.Rewind;
+                case MediaControlPlaybackCommand.Toggle: return MediaControllerNativePlaybackAction.Toggle;
             }
-            return MediaControllerPlaybackCode.Play;
+            return MediaControllerNativePlaybackAction.Play;
         }
 
-        internal static NativeRepeatMode ToNative(this MediaControlRepeatMode mode)
+        internal static MediaControllerNativeRepeatMode ToNative(this MediaControlRepeatMode mode)
         {
             Debug.Assert(Enum.IsDefined(typeof(MediaControlRepeatMode), mode));
 
-            return mode == MediaControlRepeatMode.Off ? NativeRepeatMode.On : NativeRepeatMode.Off;
+            return mode == MediaControlRepeatMode.Off ? MediaControllerNativeRepeatMode.On :
```

Comment:
You can use casting because these are same value
``` c#
 return (MediaControllerNativeRepeatMode )mode;
```

Reply:
Native and C# repeatmode are implemented differently. I don't know why but it' done already like that.
```
public enum MediaControlRepeatMode
    {
        Off,
        On,
        OneMedia
    }
```
```
internal enum MediaControllerNativeRepeatMode
    {
        On,
        Off,
        OneMedia
    }
```